types/src/transaction/mod.rs

```diff
@@ -1919,10 +1891,13 @@ impl Transaction {
         }
     }
 
-    pub fn format_for_client(&self, get_transaction_name: impl Fn(&[u8]) -> String) -> String {
-        match self {
+    pub fn format_for_client(
```

Comment:
@msmouse do you know where is this used? Changing the result type to `Result`, is this ok?